elixir/apps/web/lib/web/live/sign_in/success.ex

```diff
@@ -0,0 +1,58 @@
+defmodule Web.SignIn.Success do
+  use Web, {:live_view, layout: {Web.Layouts, :public}}
+
+  def mount(params, _session, socket) do
+    if connected?(socket) do
+      delay = Domain.Config.fetch_env!(:web, :client_redirect_delay)
+      Process.send_after(self(), :redirect_client, delay)
+    end
+
+    query_params =
+      params
+      |> Map.take(
+        ~w[fragment state actor_name account_slug account_name identity_provider_identifier]
```

Comment:
We have account properties already assigned so no need to receive them in params.

Reply:
I think I'm missing something....Where are the account properties already assigned?

Reply:
They are assigned in live view mount in the router

Reply:
From everything I've tested, it doesn't appear that account is set by the time it reaches this `mount` function.  Want to sync up briefly tomorrow to see if I'm missing something?